collector/filesystem_freebsd.go

```diff
@@ -21,78 +21,65 @@ import (
 import "C"
 
 const (
-	filesystemSubsystem = "filesystem"
+	subsystem = "filesystem"
```

Comment:
Ahh, I thought that names starting with lowercase had only local scopes. Will fix.